google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java

```diff
@@ -67,13 +66,11 @@ public String getDisplayString() {
           @Override
           public OSProcessHandler createProcessHandler(
               String workingDirectory, Map<String, String> envVariables) throws ExecutionException {
-            CloudSdkService sdkService = CloudSdkService.getInstance();
 
-            if (sdkService.getSdkHomePath() == null
-                || sdkService.getSdkHomePath().toString().isEmpty()) {
+            CloudSdkService sdkService = CloudSdkService.getInstance();
+            if (sdkService.getSdkHomePath() == null) {
               throw new ExecutionException(
-                  CloudSdkPanel.createErrorMessageWithLink(
-                      GctBundle.message("appengine.cloudsdk.location.missing.message")));
+                  GctBundle.message("appengine.run.server.sdk.misconfigured.message"));
```

Comment:
I did try clicking the link and it didn't work. Regardless of that though - should we maybe make the message more generic then "application server above"? Guide them generically to fix the application server. This gets rid of the duplication of the download link (they'll eventually see it again once they configure the app server) and it moves the download message closer to the correct source.